block/aggregation_test.go

```diff
@@ -36,7 +36,7 @@ func TestAggregationLoop_Normal_BasicInterval(t *testing.T) {
 	mockExec := mocks.NewMockExecutor(t)
 	mockSeq := mocks.NewMockSequencer(t)
 	mockDAC := mocks.NewMockDA(t)
-	logger := logging.Logger("test")
```

Comment:
ipfs uses zap under the hood, do we need to replace it throughout the repo? https://github.com/ipfs/go-log/blob/master/core.go